src/pages/Portfolio/ImpactDashboard/ImpactDashboardPage.vue

```diff
@@ -127,14 +128,15 @@ export default {
 	created() {
 		const portfolioQueryData = this.apollo.readQuery({ query: portfolioQuery });
 		const userData = portfolioQueryData?.my ?? {};
+		this.loansCount = userData.lender?.loanCount ?? 0;
```

Comment:
Wondering if this differ from `my.loans.totalCount`
@mcstover do you know?

Reply:
Thanks @mcstover!
@christian14b said that, probably we should change it to match with loans we are showing in the carousel.